core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalAppender.java

```diff
@@ -65,21 +61,18 @@ public RaftJournalAppender(RaftServer server, Supplier<RaftClient> clientSupplie
   /**
    * Sends a request to raft server asynchronously.
    * @param message the message to send
-   * @param timeout the time duration to wait before giving up on the request
    * @return a future of the server reply
    * @throws IOException if an exception occured while sending the request
    */
-  public CompletableFuture<RaftClientReply> sendAsync(Message message,
-      TimeDuration timeout) throws IOException {
```

Comment:
why is this gone?

Reply:
This PR will indeed create one new `RaftClient` per message.

I removed the `timeout` parameter from this function because it was unused in the function body. Moreover, all the callers of `sendAsync` use `.get(<timeout>)` to regulated their own timeouts when receiving a future. This parameter was therefore redundant and unused, so I removed it.

Reply:
got it. thanks.